### PR TITLE
Add Tendermint 0.34 support

### LINKED ISSUE #344: Add Tendermint 0.34 client support
As of nightly-2020-08-05, Cosmos SDK is still on Tendermint 0.33 (see https://github.com/cosmos/cosmos-sdk/blob/392121e/go.mod#L40). Upgrading to Tendermint 0.34 is planned for the stargate release, but not yet done: https://github.com/cosmos/cosmos-sdk/issues/6365.

We can prepare by creating a Tendermint 0.34 client once there is a release (check https://github.com/tendermint/tendermint/releases, https://hub.docker.com/r/tendermint/tendermint/tags/?page=1&name=v0.34)

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #344

### CODE REVIEW

File: packages/tendermint-rpc/src/config.spec.ts [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+  readonly appVersion: number;
+}
+
 export interface TendermintInstance {
   readonly url: string;
   readonly version: string;
-  readonly appCreator: string;
+  /** Values we expect in the backend */
+  readonly expected: ExpectedValues;
+  /** A workaround for https://github.com/tendermint/tendermint/issues/4489 */
```
Comment (willclarktech) [inline packages/tendermint-rpc/src/config.spec.ts:13]: 😬

**Review by willclarktech — APPROVED** [review]
LGTM.

File: packages/tendermint-rpc/src/config.spec.ts (hunk 1 above)
Comment (webmaster128) [reply packages/tendermint-rpc/src/config.spec.ts]: Fortunately the clock issue is relsolved now

**Comment by webmaster128** [comment]
Tendermint 0.34 is not starting in the CI for some strange reason. Will investigate.

**Pull request merged**